src/XrdSys/XrdSysE2T.cc

```diff
@@ -74,6 +74,11 @@ int initErrTable()
       if (EBADE - ERRNOBASE > lastGood)
          lastGood = EBADE - ERRNOBASE;
    }
+   else {
+      e2sMutex.Lock();
```

Comment:
While this is innocous, it is not needed. This code is running single-threaded during static initialization. See the file-level code containing
'int maxErrno = initErrTable();'.

Reply:
Lock/Unlock removed.